### PR TITLE
[IMP] Simpler placeholders in email templates

### PR DESCRIPTION
**Pull request by andreparames**
refs #28441

### CODE REVIEW

File: odoo_addons/mozaik_communication/wizard/distribution_list_mass_function_view.xml [hunk 1]
```diff
@@ -68,6 +68,17 @@
                                 </group>
                             </group>
                         </page>
+                        <page name="configured_placeholders" string="Configured Placeholders"
```
Comment (Olivier-LAURENT) [inline odoo_addons/mozaik_communication/wizard/distribution_list_mass_function_view.xml]: @andreparames 
This new tab forces the end-user to switch from one tab to another just to make the copy/paste. It is maybe simpler for him to view both the html body of his mail and your placeholder mechanism in the same tab.

File: odoo_addons/mozaik_communication/wizard/distribution_list_mass_function_v8.py [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
+    # Fake field for auto-completing placeholder
+    placeholder_id = fields.Many2one(
+        'email.template.placeholder', string="Placeholder",
+        domain=[('model_id', '=', 'email.coordinate')])
+    placeholder_value = fields.Char(
+        'Placeholder Value')
+
+    @api.onchange('placeholder_id')
+    def _onchange_placeholder_id(self):
+        for tmpl in self:
```
Comment (Olivier-LAURENT) [inline odoo_addons/mozaik_communication/wizard/distribution_list_mass_function_v8.py]: @andreparames 
Is the `tmpl`  name a good choice here? Self is in fact a wizard.

File: odoo_addons/mozaik_communication/wizard/distribution_list_mass_function_v8.py [hunk 3]
```diff
@@ -1,12 +1,26 @@
...
 
 class DistributionListMassFunction(models.TransientModel):
     _inherit = 'distribution.list.mass.function'
 
+    # Fake field for auto-completing placeholder
+    placeholder_id = fields.Many2one(
+        'email.template.placeholder', string="Placeholder",
+        domain=[('model_id', '=', 'email.coordinate')])
+    placeholder_value = fields.Char(
+        'Placeholder Value')
```
Comment (Olivier-LAURENT) [inline odoo_addons/mozaik_communication/wizard/distribution_list_mass_function_v8.py]: @andreparames 
Maybe a help property field is here useful for the end-user. Or instead, a more specific < p >explanation< /p > in the form of the wizard.

File: odoo_addons/mozaik_communication/wizard/distribution_list_mass_function_view.xml [hunk 4]
```diff
@@ -68,6 +68,17 @@
...
+                            attrs="{'invisible':['|',('trg_model','!=','email.coordinate'),
+                                                       ('e_mass_function','!=','email_coordinate_id')]}"
+                            groups="mozaik_base.mozaik_res_groups_user">
+                            <group>
+                                <field name="placeholder_id"
+                                       domain="[('model_id','=','email.coordinate')]"
```
Comment (Olivier-LAURENT) [inline odoo_addons/mozaik_communication/wizard/distribution_list_mass_function_view.xml]: @andreparames 
Must the domain be re-specified here ?

File: odoo_addons/mozaik_communication/wizard/distribution_list_mass_function_v8.py (hunk 3 above)
Comment (Olivier-LAURENT) [inline odoo_addons/mozaik_communication/wizard/distribution_list_mass_function_v8.py]: The string property is not really necessary here I think.

File: odoo_addons/mozaik_communication/__openerp__.py [hunk 5]
```diff
@@ -34,6 +34,8 @@
         'mozaik_person',
         'mozaik_membership',
         'mozaik_retrocession',
+        # from https://github.com/OCA/crm
```
Comment (Olivier-LAURENT) [inline odoo_addons/mozaik_communication/__openerp__.py]: @andreparames 
Here is just a question/suggestion: would the "social" repo be a better place for this new feature ?

Comment (andreparames) [reply odoo_addons/mozaik_communication/__openerp__.py]: Good question. OCA/crm says it's for "CRM, Mail & Newsletter", but OCA/social seems to already have a module related to email templates. I'll ask in the PR.

Comment (andreparames) [reply odoo_addons/mozaik_communication/__openerp__.py]: I've moved the PR to OCA/social.

**Comment by andreparames** [comment]
Thanks for the review! I've made the suggested changes.

**Review by Olivier-LAURENT — APPROVED** [review]
@andreparames 
Thx for the job

**Pull request merged**